keybot/main.go

```diff
@@ -4,12 +4,74 @@
 package main
 
 import (
+	"bytes"
 	"log"
 	"os"
 
 	"github.com/keybase/slackbot"
+	"gopkg.in/alecthomas/kingpin.v2"
 )
 
+func setEnv(name string, val string) error {
+	_, err := slackbot.NewExecCommand("/bin/launchctl", []string{"setenv", name, val}, false, "Set the env").Run([]string{})
+	return err
+}
+
+func kingpinHandler(args []string) (string, error) {
+	app := kingpin.New("slackbot", "Command parser for slackbot")
+	app.Terminate(nil)
+	stringBuffer := new(bytes.Buffer)
+	app.Writer(stringBuffer)
+
+	build := app.Command("build", "Build things")
+
+	clientCommit := build.Flag("client-commit", "Build a specific client commit hash").String()
```

Comment:
I would make these flags on buildPlease and buildTest separately

So..

```
buildPleaseClientCommit := buildPlease.Flag("client-commit", "Build a specific client commit hash").String()
...

buildTestClientCommit := buildTest.Flag("client-commit", "Build a specific client commit hash").String()
```

I know it's duplicating code but the flags should be on the subcommands?